File: sentimental-mario-less/mario.py
from cs50 import get_int

h = get_int("Height: ")
while(h < 1 or h > 8):
    h = get_int("Height: ")

for i in range(h):
    for j in range(h):
        if (h-i-j) > 1:
            print(" ", end="")
        else:
            print("#", end="")
    print()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1);
    for (int i = 0; i < h; i++)
    {
        for (int j = 0; j < h; j++)
        {
            if (h - i - j > 1)
            {
                printf("%c", ' ');
            }
            else
            {
                printf("%c", '#');
            }
        }
        printf("\n");
    }
}
